test/test_database.py

```diff
@@ -12,12 +12,16 @@
 TEST_DIR = Path(os.path.abspath(os.path.dirname(__file__)))
 PROJECT_ROOT = TEST_DIR.parent
 sys.path.insert(0, str(PROJECT_ROOT))
-from enclave_wrangler.objects_api import find_and_add_missing_csets_to_db, get_bidirectional_csets_sets
+from enclave_wrangler.objects_api import get_bidirectional_csets_sets
 
-# TODO: temp
-# Note: Ad hoc import: On 2023/09/12, we found 93 missing csets. The below code was used to import them.
-if __name__ == '__main__':
-    find_and_add_missing_csets_to_db()
+FAIL_MSG = (
```

Comment:
## Test failure message
If test fails, gives some idea as to why it might have failed, what csets were missing, and how to fix the issue in the interim.